src/host/ntprivapi.cpp

```diff
@@ -13,12 +13,12 @@
 
     CLIENT_ID ClientId;
     ClientId.UniqueProcess = UlongToHandle(*ProcessId);
-    ClientId.UniqueThread = 0;
+    ClientId.UniqueThread = nullptr;
 
     HANDLE ProcessHandle;
     NTSTATUS Status = s_NtOpenProcess(&ProcessHandle, PROCESS_QUERY_LIMITED_INFORMATION, &oa, &ClientId);
 
-    PROCESS_BASIC_INFORMATION BasicInfo = { 0 };
+    PROCESS_BASIC_INFORMATION BasicInfo = {};

```

Comment:
This mildly concerns me - we've ran into insane bugs in the past where not zero-initializing something has resulted in bugs. 
```suggestion
    PROCESS_BASIC_INFORMATION BasicInfo = { 0 };
```